### PR TITLE
Refactor UIEdgeInsets image flip logic

### PR DESCRIPTION
**Pull request by matthusby**
Make the function on UIEdgeInset private so all calls go through UIButton that is aware of the language direction.

https://github.com/wordpress-mobile/WordPress-iOS/pull/8253 is required for this as it removed one call to the function that is now private.

### CODE REVIEW

**Comment by elibud** [comment]
This one was fixed here: https://github.com/wordpress-mobile/WordPress-iOS/pull/8824

**Comment by elibud** [comment]
Reopening, I confused this one with: https://github.com/wordpress-mobile/WordPress-iOS/pull/8253

**Comment by elibud** [comment]
@etoledom since you've been working on RTL would you mind taking a look at this one? Thanks

File: WordPress/Classes/ViewRelated/Post/PostCardActionBar.m [hunk 1]
```diff
@@ -315,9 +315,8 @@ - (PostCardActionBarItem *)moreItem:(BOOL)isLastBatch;
                                    highlightedImage:nil];
     } else {
         item = [PostCardActionBarItem itemWithTitle:NSLocalizedString(@"More", @"")
-                                              image:[UIImage imageNamed:@"icon-post-actionbar-more"]
+                                              image:[[UIImage imageNamed:@"icon-post-actionbar-more"] imageFlippedForRightToLeftLayoutDirection]
```
Comment (etoledom) [inline WordPress/Classes/ViewRelated/Post/PostCardActionBar.m]: This image is symmetrical, right? Probably it won't be necessary to flip it.

**Comment by elibud** [comment]
Closing due to inactivity.

**Pull request closed without merging**